app/controllers/oauth_api_gouv_logins_controller.rb

```diff
@@ -1,7 +1,46 @@
 class OAuthAPIGouvLoginsController < ApplicationController
+  skip_before_action :require_login
+
   def new
+    redirect_current_user_to_homepage if current_user
```

Comment:
Toujours dans la même logique j'introduirais un `user_signed_in?` ici

https://github.com/heartcombo/devise#controller-filters-and-helpers